src/Aspire.Hosting.Elasticsearch/ElasticsearchContainerImageTags.cs

```diff
@@ -7,6 +7,6 @@ internal static class ElasticsearchContainerImageTags
 {
     public const string Registry = "docker.io";
     public const string Image = "library/elasticsearch";
-    public const string Tag = "8.15.0";
+    public const string Tag = "8.15.1";
```

Comment:
@eerhardt @radical we need to update the feed, right? I can do it if you say yes

Reply:
Correct, we need to update the registry. I was going to take care of it but if you're planning on doing it then that works for me too.